From 8604ad92dd1e3e4f766b9ff23008dfff4853c251 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Sat, 30 Mar 2019 01:54:27 +0000
Subject: [PATCH] OS-7327 Want connstat(1M) command to display per-connection
 TCP statistics (fix timestamp fallthrough) Reviewed by: Rob Johnston
 <rob.johnston@joyent.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Approved by: Patrick Mooney
 <patrick.mooney@joyent.com>

---
 usr/src/cmd/connstat/connstat_main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/usr/src/cmd/connstat/connstat_main.c b/usr/src/cmd/connstat/connstat_main.c
index 8fa8dc4fb7..dd58e2ac2b 100644
--- a/usr/src/cmd/connstat/connstat_main.c
+++ b/usr/src/cmd/connstat/connstat_main.c
@@ -461,6 +461,7 @@ print_timestamp(timestamp_fmt_t timestamp_fmt, boolean_t parsable)
 		if (len > 0) {
 			(void) printf("%s%s\n", pfx, dstr);
 		}
+		break;
 	}
 	default:
 		abort();
-- 
2.21.0

